lte/gateway/c/session_manager/SessionEvents.cpp

```diff
@@ -213,14 +213,14 @@ void EventsReporterImpl::session_terminated(
   event.set_event_type(SESSION_TERMINATED_EV);
   event.set_tag(imsi);
 
-  folly::dynamic event_value           = folly::dynamic::object;
-  event_value[IMSI]                    = imsi;
-  event_value[IP_ADDR]                 = session_cfg.common_context.ue_ipv4();
-  event_value[IPV6_ADDR]               = session_cfg.common_context.ue_ipv6();
-  event_value[MSISDN]                  = session_cfg.common_context.msisdn();
-  event_value[APN]                     = session_cfg.common_context.apn();
-  event_value[SESSION_ID]              = session->get_session_id();
-  SessionState::TotalCreditUsage usage = session->get_total_credit_usage();
+  folly::dynamic event_value            = folly::dynamic::object;
+  event_value[IMSI]                     = imsi;
+  event_value[IP_ADDR]                  = session_cfg.common_context.ue_ipv4();
+  event_value[IPV6_ADDR]                = session_cfg.common_context.ue_ipv6();
+  event_value[MSISDN]                   = session_cfg.common_context.msisdn();
+  event_value[APN]                      = session_cfg.common_context.apn();
+  event_value[SESSION_ID]               = session->get_session_id();
+  SessionCredit::TotalCreditUsage usage = session->get_total_credit_usage();
```

Comment:
This is just cosmetic, but is there any reason why we can't put all the event_values togheter and separately, any other modification we do when we execute this function?

I think it will be help in the future if we want to see what is part of the event and what it is not

```
...
  event_value[IP_ADDR]        = session_cfg.common_context.ue_ipv4();
  event_value[IPV6_ADDR]      = session_cfg.common_context.ue_ipv6();
  event_value[MSISDN]         = session_cfg.common_context.msisdn();
  event_value[TOTAL_TX]       = usage.charging_tx + usage.monitoring_tx;
  event_value[TOTAL_RX]       = usage.charging_rx + usage.monitoring_rx;
  event_value[CHARGING_TX]    = usage.charging_tx;

  SessionCredit::TotalCreditUsage usage = session->get_total_credit_usage();
  const auto start_time       			= session->get_pdp_start_time();
  const auto end_time         			= session->get_pdp_end_time();


```